controllers/smbshare_controller.go

```diff
@@ -83,6 +83,8 @@ func (r *SmbShareReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&sambaoperatorv1alpha1.SmbShare{}).
 		Owns(&corev1.PersistentVolumeClaim{}).
+		Owns(&corev1.ConfigMap{}).
 		Owns(&appsv1.Deployment{}).
+		Owns(&appsv1.StatefulSet{}).
```

Comment:
I don't remember the discussion outcome but what about `corev1.Service`?

Reply:
Sure, anything resource type w generate from an SmbShare is fair game (although not required).